### ISSUE TITLE
Some questions about large training loss

### ISSUE THREAD

**Issue by lly2111101** [pkuCactus/BDCN#29]
Hi, I download you code and BSDS500 dataset to learn edge detection model and print the loss of each iteration, however, the training losses are very large and keep increasing which indicate that the model is not converge. I just keep the default parameters, can you give me any hints to solve these problems?

**Comment by wuwou** [comment]
I have the same question with you. and when i train the model, training loss becomes very large which fluctuates around 4000.

**Comment by Takugo** [comment]
I also found that there were increasing loss result and it cannot coverage, what about you guys setting of `parser.add_argument()` ?

**Comment by ahpu2014** [comment]
> 我和你有同样的问题。当我训练模型时，训练损失变得非常大，在 4000 左右波动。



> 我和你有同样的问题。当我训练模型时，训练损失变得非常大，在 4000 左右波动。

请问大佬最终的loss值稳定在哪里？我这边在15000左右

**Comment by Takugo** [comment]
This ain't a simple issue.

**Comment by ahpu2014** [comment]
> 这不是一个简单的问题。

请问您这边的效果大致是什么样的？可以留一个联系方式，我们交流一下吗？1593662206@qq.com

**Comment by wenya1994** [comment]
> Hi, I download you code and BSDS500 dataset to learn edge detection model and print the loss of each iteration, however, the training losses are very large and keep increasing which indicate that the model is not converge. I just keep the default parameters, can you give me any hints to solve these problems?

问题怎么解决的呀，求指点

**Comment by Takugo** [comment]
> > Hi, I download you code and BSDS500 dataset to learn edge detection model and print the loss of each iteration, however, the training losses are very large and keep increasing which indicate that the model is not converge. I just keep the default parameters, can you give me any hints to solve these problems?
> 
> 问题怎么解决的呀，求指点

The issus is hanging there for two years, and I have left this part.
